docs/dev-guide.md

```diff
@@ -471,10 +322,7 @@ WHERE
 - For queries where it would be an error to not return exactly one result row:
 
   ```javascript
-  sqldb.queryOneRow(sql.block_name, params, function (err, result) {
-    if (ERR(err, callback)) return;
-    var obj = result.rows[0]; // guaranteed to exist and no more
-  });
+  const question = await queryOneRow(sql.block_name, {}, QuestionSchema);
```

Comment:
If the params argument is empty on purpose, maybe skip it? Otherwise, maybe add a comment that parameters may be added.

Reply:
Fixed in 7ec3678.